src/pool/srv_cli.c

```diff
@@ -75,8 +78,9 @@ dsc_pool_open(uuid_t pool_uuid, uuid_t poh_uuid, unsigned int flags,
 	D_DEBUG(DF_DSMC, DF_UUID": create: hdl="DF_UUIDF" flags=%x\n",
 		DP_UUID(pool_uuid), DP_UUID(pool->dp_pool_hdl), flags);
 
-	dc_pool_hdl_link(pool);
-	dc_pool2hdl(pool, ph);
+	dc_pool_hdl_link(pool); /* +1 ref */
+	dc_pool2hdl(pool, ph);  /* +1 ref */
+
 out:
```

Comment:
Not from this patch, I'm wondering why we have L44-48
```
	if (daos_handle_is_valid(*ph)) {
		pool = dc_hdl2pool(*ph);
		if (pool != NULL)
			D_GOTO(out, rc = 0);
	}
```

Reply:
Hmm. My understanding is to check if this handle was already opened before. But this is not used now in sources. Probably we should remove this in next patch to avoid incorrect behavior if *ph will contain a garbage (handle is valid if it not NULL).